id-library-showcase/README.md

```diff
@@ -0,0 +1,120 @@
+# ID Library Showcase
+Here we demonstrate how to prove and verify properties about an identity behind an account, such as
+- reveal an attribute,
+- prove ownership of an account,
+- prove that an attribute is in a range.
+
+## Account key structure
+An account is owned by one or more credential holders. Each credential holder has a `credential index` and its own set of keys used to sign transactions. Each account has a threshold indicating how many of the credential holders owning the account is needed for signing a transaction. The account keys of an account consist of all the keys of all the credential holders. When creating an account, the user creating it will be the only account owner and will always have credential index `0`. When a user - a credential holder of an account - wants to prove or reveal anything about the account, it is always with respect to its own credential index.
```

Comment:
```suggestion
An account is owned by one or more credential holders. Each credential holder has a `credential index` and its own set of keys used to sign transactions. Each account has a threshold indicating how many of the credential holders owning the account are needed for signing a transaction. The account keys of an account consist of all the keys of all the credential holders. When creating an account, the user creating it will be the only account owner and will always have credential index `0`. When a user - a credential holder of an account - wants to prove or reveal anything about the account, it is always with respect to its own credential index.
```